pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -271,27 +271,24 @@ func (r *bindingReconciler) reconcile(ctx context.Context, apiBinding *apisv1alp
 			return reconcileStatusStopAndRequeue, err
 		}
 
-		// If there are multiple versions, there must be an APIConversion
-		if len(schema.Spec.Versions) > 1 {
-			if _, err := r.getAPIConversion(logicalcluster.From(schema), schema.Name); err != nil {
-				// Need to wait until the APIConversion is present before we can proceed to create the bound CRD
-				conditions.MarkFalse(
-					apiBinding,
-					apisv1alpha1.APIExportValid,
-					apisv1alpha1.InternalErrorReason,
-					conditionsv1alpha1.ConditionSeverityError,
-					"Invalid APIExport. Please contact the APIExport owner to resolve",
-				)
+		// If there are multiple versions, the conversion strategy must be defined in the APIResourceSchema
+		if len(schema.Spec.Versions) > 1 && schema.Spec.Conversion == nil {
```

Comment:
Technically, witohut APIConversions we need this. This condition only exists because APIConversions are asynchronously replicated and it might be that the controller has to wait a bit until it propagates through the cache server.

Reply:
So, this is fine right considering APIConversions support doesn't exist anymore? And there's nothing for the cache server to catch up to.

Reply:
correct.